test/unit/host_managed_extensions_test.rb

```diff
@@ -18,7 +18,7 @@ def test_update_kernel_version
       package_json = { :name => 'kernel', :version => '1',
                        :release => '1.el7', :arch => 'x86_64', :epoch => '1',
                        :nvra => 'kernel-1-1.el7.x86_64' }
-      host.import_package_profile([::Katello::Pulp::SimplePackage.new(package_json)])
+      host.import_package_profile([::Katello::SimplePackage.new(package_json)])
```

Comment:
Do I need to release a new version after that ?

Reply:
Only if you want to be compatible with Katello 4.8+ :)

No, seriously, given I had to edit `app/models/concerns/foreman_kernel_care/host_managed_extensions.rb` it indicates that the current release is not compatible with current Katello releases, so I think a release of kernel_care should happen.

That said, kernel_care being broken doesn't block anything (besides k_c itself), so it's totally up to you when you have time for a release :)

Reply:
Okay got it. Thanks